apps/www/app/code-examples.tsx

```
@@ -1,7 +1,9 @@
-"use client";
-import { Editor } from "@/components/analytics/analytics-bento";
+"use client";;
 import { PrimaryButton, SecondaryButton } from "@/components/button";
 import { SectionTitle } from "@/components/section";
+import { CurlIcon, ElixirIcon, GoIcon, JavaIcon, LangIconProps, PythonIcon, RustIcon, TSIcon } from "@/components/svg/lang-icons";
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)